pkg/cli/root.go

```diff
@@ -164,6 +164,14 @@ func LoadConfiguration(config *api.Config, configPath string) {
 		panic(errors.ErrBadConfig)
 	}
 
+	// check authorization config, it should have basic auth enabled or ldap
+	if config.HTTP.RawAccessControl != nil {
+		if config.HTTP.Auth == nil || (config.HTTP.Auth.HTPasswd.Path == "" && config.HTTP.Auth.LDAP == nil) {
+			log.Error().Err(errors.ErrBadConfig).Msg("Bad configuration, authorization config without BASIC auth or LDAP")
```

Comment:
Msg("access control config requires htpassd or ldap authentication to be enabled")